src/axom/sidre/examples/sidre_generateindex.cpp

```diff
@@ -354,16 +355,27 @@ void generate_multidomain_blueprint(sidre::DataStore* ds,
 
   std::string holder_name = "domain_data";
   std::string mesh_name = "mesh";
-  sidre::Group* holder = ds->getRoot()->createGroup(holder_name);
+  sidre::Group* holder = ds->getRoot()->createGroup(holder_name, use_list);
 
   std::string domain_pattern = "domain_{domain:06d}";
   ds->getRoot()->createViewString("domain_pattern", domain_pattern);
 
   for(int64_t i = domain_begin; i < domain_end; ++i)
   {
-    std::string domain_name = axom::fmt::format("domain_{:06d}", i);
+    sidre::Group* mroot = 0;
```

Comment:
Should we use `nullptr` for better readability?

Reply:
yes